SocialCareCaseViewerApi/V1/Domain/RelatedRelationship.cs

```diff
@@ -0,0 +1,14 @@
+namespace SocialCareCaseViewerApi.V1.Domain
+{
+    public class RelatedRelationship
+    {
+        public long Id { get; set; }
+        public long PersonId { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Gender { get; set; }
+        public string IsMainCarer { get; set; }
+        public string IsInformalCarer { get; set; }
+        public string Details { get; set; }
```

Comment:
Details can be null right?

Can any of the other properties on RelatedRelationship.

I'd advise being explicit about what can and can't be null (https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references)

Reply:
Yes, Details can be null!

I'll set it nullable and add the #nullable enable annotation right on top of it